### DISCUSSION TITLE
Tutorial on Query Loop block variations

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#34] [Topic Ideas]
I'd like to put together a foundational tutorial that covers building custom Query Loop variations for plugin authors.  At its most basic, this can be something as simple as using the built-in allowed controls and inner blocks for the template.  

However, it's probably worth exploring some basic post metadata inclusion with a custom settings control for filtering the query.  For example, if showing a "book reviews" variation, the tutorial could also teach how to filter those posts by a "star rating" custom field.

I have most of this mapped out in my mind at this point, but any feedback on what anyone would like to see the tutorial cover is welcome.

**Comment by bph** [comment]
Thanks for taking this on @justintadlock - Locked the discussion for now. We can always open a new post, after your post is published to collect more aspects for the Query block
